locale/en/blog/release/v4.0.0.md

```diff
@@ -0,0 +1,152 @@
+* [[`f146f686b7`](https://github.com/nodejs/node/commit/f146f686b7)] - **(SEMVER-MAJOR)** **test**: fix test-repl-tab-complete.js for V8 4.5 (Ali Ijaz Sheikh) [#2509](https://github.com/nodejs/node/pull/2509)
+* [[`fe4b309fd3`](https://github.com/nodejs/node/commit/fe4b309fd3)] - **test**: refactor to eliminate flaky test (Rich Trott) [#2609](https://github.com/nodejs/node/pull/2609)
+* [[`619721e6b8`](https://github.com/nodejs/node/commit/619721e6b8)] - **test**: mark eval_messages as flaky (Alexis Campailla) [#2648](https://github.com/nodejs/node/pull/2648)
+* [[`93ba585b66`](https://github.com/nodejs/node/commit/93ba585b66)] - **test**: mark test-vm-syntax-error-stderr as flaky (João Reis) [#2662](https://github.com/nodejs/node/pull/2662)
+* [[`367140bca0`](https://github.com/nodejs/node/commit/367140bca0)] - **test**: mark test-repl-persistent-history as flaky (João Reis) [#2659](https://github.com/nodejs/node/pull/2659)
+* [[`f6b093343d`](https://github.com/nodejs/node/commit/f6b093343d)] - **timers**: minor `_unrefActive` fixes and improvements (Jeremiah Senkpiel) [#2540](https://github.com/nodejs/node/pull/2540)
+* [[`403d7ee7d1`](https://github.com/nodejs/node/commit/403d7ee7d1)] - **timers**: don't mutate unref list while iterating it (Julien Gilli) [#2540](https://github.com/nodejs/node/pull/2540)
+* [[`7a8c3e08c3`](https://github.com/nodejs/node/commit/7a8c3e08c3)] - **timers**: Avoid linear scan in `_unrefActive`. (Julien Gilli) [#2540](https://github.com/nodejs/node/pull/2540)
+* [[`b630ebaf43`](https://github.com/nodejs/node/commit/b630ebaf43)] - **win,msi**: Upgrade from old upgrade code (João Reis) [#2439](https://github.com/nodejs/node/pull/2439)
+
+
+Windows 32-bit Installer: http://nodejs.org/dist/v4.0.0/node-v4.0.0-x86.msi
+
+Windows 64-bit Installer: http://nodejs.org/dist/v4.0.0/node-v4.0.0-x64.msi
+
+Mac OS X 64-bit Installer: http://nodejs.org/dist/v4.0.0/node-v4.0.0.pkg
+
+Mac OS X 64-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-darwin-x64.tar.gz
+
+Linux 32-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-linux-x86.tar.gz
+
+Linux 64-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-linux-x64.tar.gz
+
+SunOS 32-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-sunos-x86.tar.gz
+
+SunOS 64-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-sunos-x64.tar.gz
+
+ARMv6 32-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-linux-armv6l.tar.gz
+
+ARMv7 32-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-linux-armv7l.tar.gz
+
+ARMv8 64-bit Binary: http://nodejs.org/dist/v4.0.0/node-v4.0.0-linux-arm64.tar.gz
+
+Source Code: http://nodejs.org/dist/v4.0.0/node-v4.0.0.tar.gz
+
+Other release files: http://nodejs.org/dist/v4.0.0/
+
+Website: http://nodejs.org/docs/v4.0.0/
+
+Documentation: http://nodejs.org/docs/v4.0.0/api/
+
+Shasums:
+```
+{{shasums}}
```

Comment:
The list of release files and their corresponding shasums are missing, e.g. for v3.3.0: https://iojs.org/dist/v3.3.0/SHASUMS256.txt.asc

For all I know you havent done the final build yet?